website/sidebars.json

```diff
@@ -9,25 +9,25 @@
     "MobX core": [
       "refguide/observable",
       "refguide/action",
-      "refguide/autorun",
-      "refguide/computed"
+      "refguide/computed",
+      "refguide/autorun"
     ],
     "MobX and React": [
-      "react/react-integration"
+      "react/react-integration",
+      "react/react-performance"
     ],
     "Tips & Tricks": [
+      "best/store",
```

Comment:
I've moved it to the 1st place so there's a clear distinction between basic and advanced docs in the heading.